client/ayon_flame/plugins/publish/collect_audio.py

```diff
@@ -0,0 +1,34 @@
+import pyblish.api
+
+
+class CollectAudio(pyblish.api.InstancePlugin):
+    """Collect new audio."""
+
+    order = pyblish.api.CollectorOrder - 0.48
+    label = "Collect Audio"
+    hosts = ["flame"]
+    families = ["audio"]
+
+    def process(self, instance):
+        """
+        Args:
+            instance (pyblish.Instance): The shot instance to update.
+        """
+        # Retrieve instance data from parent instance shot instance.
+        parent_instance_id = instance.data["parent_instance_id"]
+        edit_shared_data = instance.context.data["editorialSharedData"]
+        instance.data.update(
+            edit_shared_data[parent_instance_id]
+        )
+
+        if instance.data.get("reviewTrack") is not None:
+            instance.data["reviewAudio"] = True
+            instance.data.pop("reviewTrack")
```

Comment:
```suggestion
            # Remove review track to avoid creation of reviewable
            #     for the instance
            instance.data.pop("reviewTrack")
```